src/rootfs_chooser.py

```diff
@@ -0,0 +1,19 @@
+import logging
+import requests
+
+class RootfsAccessError(Exception):
+    pass
+
+class RootfsChooser(object):
+    __ROOTFS_BASE = 'http://lava.free-electrons.com/downloads/rootfs'
+
+    def get_url(self, board):
+        rootfs = '%s/%s' % (self.__ROOTFS_BASE, board['rootfs'])
```

Comment:
It's not a simple as that, because the architecture is not specific enough. We have to keep the information that a board is simply `arm` or `arm64`, because we (and LAVA) deduce many things from that (such as the kernel image format), but for the `rootfs`, we need to have more specific informations such as the `arm` version. It would just complicate code for nothing to store only the specific `arm` version, dropping thus the general architecture, because we would end up with some switch cases retrieving the lost information to build the job.